script/seed.js

```diff
@@ -1,29 +1,88 @@
-'use strict'
+"use strict";
 
-const {db, models: {User} } = require('../server/db')
+const {
+  db,
+  models: { User, Project, Section, File, user_projects },
+} = require("../server/db");
 
 /**
  * seed - this function clears the database, updates tables to
  *      match the models, and populates the database.
  */
 async function seed() {
-  await db.sync({ force: true }) // clears db and matches models to tables
-  console.log('db synced!')
+  await db.sync({ force: true }); // clears db and matches models to tables
+  console.log("db synced!");
 
-  // Creating Users
   const users = await Promise.all([
-    User.create({ username: 'cody', password: '123' }),
-    User.create({ username: 'murphy', password: '123' }),
-  ])
+    User.create({ username: "robert", password: "123" }),
+    User.create({ username: "philip", password: "123" }),
+    User.create({ username: "chris", password: "123" }),
+  ]);
 
-  console.log(`seeded ${users.length} users`)
-  console.log(`seeded successfully`)
+  const projects = await Promise.all([
+    Project.create({ name: "project1" }),
+    Project.create({ name: "project2" }),
+  ]);
+
+  const sections = await Promise.all([
+    Section.create({ projectId: 1, sectionNumber: 1 }),
+    Section.create({ projectId: 1, sectionNumber: 2 }),
+    Section.create({ projectId: 1, sectionNumber: 3 }),
+    Section.create({ projectId: 2, sectionNumber: 1 }),
+    Section.create({ projectId: 2, sectionNumber: 2 }),
+  ]);
+
+  const files = await Promise.all([
+    File.create({
+      name: "file1",
+      filePath: "Cigarettes-In-Hell.ogg",
+      type: "ogg",
+      sectionId: 1,
+      userId: 1,
+    }),
+    File.create({
+      name: "file2",
+      filePath: "Cigarettes-In-Hell.ogg",
+      type: "ogg",
+      sectionId: 1,
+      userId: 1,
+    }),
+    File.create({
+      name: "file3",
+      filePath: "Cigarettes-In-Hell.ogg",
+      type: "ogg",
+      sectionId: 2,
+      userId: 1,
+    }),
+    File.create({
+      name: "file4",
+      filePath: "Cigarettes-In-Hell.ogg",
+      type: "ogg",
+      sectionId: 3,
+      userId: 1,
+    }),
+    File.create({
+      name: "file5",
+      filePath: "Cigarettes-In-Hell.ogg",
+      type: "ogg",
+      sectionId: 4,
+      userId: 1,
+    }),
+  ]);
+
+  console.log(
+    `seeded ${users.length} users, ${projects.length} projects, ${sections.length} sections, and ${files.length} files`
+  );
+
+  console.log(`seeded successfully`);
   return {
```

Comment:
can you delete this return? its not used and it doesnt make sense since we deleted these users